std_srvs dependency missing from package.xml

It seems that the `std_srvs` dependency is missing from the `package.xml`

This causes an error during build using catkin here is the message of the error:

```bash
Errors << witmotion_ros:check /root/catkin_ws/logs/witmotion_ros/build.check.001.log                
CMake Error at /opt/ros/noetic/share/catkin/cmake/catkin_package.cmake:196 (message):
  catkin_package() the catkin package 'std_srvs' has been find_package()-ed
  but is not listed as a build dependency in the package.xml
Call Stack (most recent call first):
  /opt/ros/noetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  CMakeLists.txt:20 (catkin_package)


make: *** [Makefile:1366: cmake_check_build_system] Error 1
cd /root/catkin_ws/build/witmotion_ros; catkin build --get-env witmotion_ros | catkin env -si  /usr/bin/make cmake_check_build_system; cd -
```

The solution is simply to add `std_srvs` to the `package.xml`:

```xml
...
<depend>std_srvs</depend>
...
```

--- a/package.xml
+++ b/package.xml
@@ -14,6 +14,7 @@
     <depend>geometry_msgs</depend>
     <depend>message_runtime</depend>
     <depend>std_msgs</depend>
+    <depend>std_srvs</depend>
     <depend>nav_core</depend>
     <depend>nav_msgs</depend>
     <depend>tf2</depend>

